specification/attributes/key_value_format.md

```diff
@@ -0,0 +1,32 @@
+# Key-Value Format
+
+This is the specified format for providing data in the form of key-value pairs, which are also known as name-value pairs, attribute-value pairs, and field-value pairs in other contexts. 
+
+This format is often used in place of enumerating data which would be inherently sparse and/or without predetermined keys (attributes) into separate columns. To faciliate parsing, this format does not support nesting.
+
```

Comment:
To be consistent with the other attributes, we should add a normative line here that says MUST meet all requirements listed below (in the requirements section)